halibot/halauth.py

```diff
@@ -1,6 +1,16 @@
 import json
 import logging
 
+def hasPermission(perm, reply=False):
```

Comment:
Maybe make the auto-reply a configuration option?

If so, where, main config, or per module/instance?

I can almost see a case where some channels may want to silence all failures, but others may still want them.